File: jeonghyeon/5week/수박수박수박수박수박수.js
function solution(n) {
    var answer = '';
    for(let i = 0; i < n; i++) { // n번 반복
        if(i % 2 === 0) { // i가 짝수
            answer += "수"; // 문자열에 "수" 추가
        } else { // i가 홀수
            answer += "박"; // 문자열에 "박" 추가
        }
    }
    return answer; // n번 반복 후 결과 문자열
}

File: dongmin/week5/수박수박수박수박수박수.java
class Solution {
    public String solution(int n) {
        String answer = "";
        for (int i = 1; i <= n; i++) {
            answer += i%2==0 ? "박" : "수";     //i가 2로 나누어 떨어지면 "박"을 그렇지 않으면 "수"를 answer에 저장한다
        }
        return answer;
    }
}